common/mqtt_client/src/lib.rs

```diff
@@ -374,12 +374,12 @@ pub struct Config {
     /// frontend operations.
     ///
     /// Default: `10`.
-    queue_capacity: usize,
+    pub queue_capacity: usize,
```

Comment:
Why do you need to change this visibility?

Reply:
When I try to update the port number in the config, the compiler throws an error saying its private, so cant change.

Reply:
Added a setter function and reverted it back to private